src/app/shared/directives/autofocus/directive.ts

```diff
@@ -27,15 +27,15 @@ export class AutofocusDirective implements AfterViewInit, OnDestroy {
   constructor(private readonly _el: ElementRef) {}
 
   ngAfterViewInit(): void {
-    if (!this.opened) {
-      throw new Error('[opened] event binding is undefined');
+    if (this.opened) {
+      this.opened.pipe(takeUntil(this._unsubscribe)).subscribe(opened => {
```

Comment:
Since it should only work when `opened` is `true` then use a `filter` pipe here to avoid `if` inside the subscribe.